src/utils/csv_writer.py

```diff
@@ -1,5 +1,8 @@
 import pandas as pd
 import os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Not needed

Reply:
We found it necessary when trying on another machine even with WSL installed. We couldn't figure out why it was needed, just that it was. If you know why though lmk too :)

Reply:
ahh the joys of the shell, no problem!

Reply:
I just tested it again without, turns out its fine, so not sure why I added it, or what changed since that commit. 

I will revert this, since in your case @Surxe, you can have a single `load_dotenv()` call in `deadbot.py` if thats the only solution

Reply:
Sure thing!